src/main/java/io/github/tryexceptelse/jdex/be/entries/LastName.java

```diff
@@ -19,4 +19,27 @@ public LastName() {
     public LastName(String entryString) {
         super(entryString);
     }
+
+    /**
+     *
+     * @param entryString: the String to be checked
+     * @return: will return False if string is empty, or just contains ' ' chars.
+     */
+    public static boolean checkStringIsValid(String entryString)
+    {
+        if(!ContactEntry.checkStringIsValid(entryString))
+           return false;
+        else if (entryString==null)
+            return false;
+        //iterates through each char of the String and checks if it is a ' '. If one of them is not, returns true
+        String str = new String(entryString);
```

Comment:
why is a new string created here which is identical to entryString? and why is a String constructor used?